### PR TITLE
Add docs for node discovery via DNS

### PR DESCRIPTION
**Pull request by phuctd95**

### CODE REVIEW

**Review by huyngopt1994 — APPROVED** [review]

File: docs/node-operators/setup/mainnet/validator.md [hunk 1]
```diff
@@ -130,6 +130,14 @@ The size of your node will also grow over time.
   RONIN_PARAMS=--http.api eth,net,web3,consortium --miner.gaslimit 100000000 --miner.gasreserve 10000000
   ```
 
+  An alternative to set the boot node is to use  node discovery via DNS (see [REP-0005](https://github.com/axieinfinity/REPs/blob/main/REP-0005.md)). In the `.env` file
```
Comment (RodinGolodin) [inline docs/node-operators/setup/mainnet/validator.md:133]: to set -> to setting

Comment (RodinGolodin) [inline docs/node-operators/setup/mainnet/validator.md:133]: In the `.env` file, make the following changes:

File: docs/node-operators/setup/mainnet/validator.md [hunk 2]
```diff
@@ -130,6 +130,14 @@ The size of your node will also grow over time.
...
+  -  Delete the `BOOTNODES=...` line
+  -  Append the `--discovery.dns enrtree://...` at the end of `RONIN_PARAMS` line
```
Comment (RodinGolodin) [inline docs/node-operators/setup/mainnet/validator.md:135]: Append `--discovery.dns enrtree://...` at the end of the `RONIN_PARAMS` line.

File: docs/node-operators/setup/mainnet/validator.md [hunk 3]
```diff
@@ -130,6 +130,14 @@ The size of your node will also grow over time.
...
+
+  We provide the DNS tree link for node operators.
```
Comment (RodinGolodin) [inline docs/node-operators/setup/mainnet/validator.md:137]: Can "DNS tree link" be replaced with "DNS node list"? The sentence will then look like this: "We provide a DNS node list for node operators:"

File: docs/node-operators/setup/mainnet/validator.md [hunk 4]
```diff
@@ -130,6 +130,14 @@ The size of your node will also grow over time.
   RONIN_PARAMS=--http.api eth,net,web3,consortium --miner.gaslimit 100000000 --miner.gasreserve 10000000
   ```
 
+  An alternative to set the boot node is to use  node discovery via DNS (see [REP-0005](https://github.com/axieinfinity/REPs/blob/main/REP-0005.md)). In the `.env` file
+  -  Delete the `BOOTNODES=...` line
+  -  Append the `--discovery.dns enrtree://...` at the end of `RONIN_PARAMS` line
+
+  We provide the DNS tree link for node operators. 
+  - Saigon Testnet: `enrtree://AJCNIAXQIPO55NW3QE2NUBBDMPYZDOQUCAEUS65NHQFMUUFES5KOW@saigon.nodes.roninchain.com`
```
Comment (RodinGolodin) [inline docs/node-operators/setup/mainnet/validator.md:138]: Add an empty line before this line.

**Review by RodinGolodin — CHANGES_REQUESTED** [review]
Please address the comments.

**Review by RodinGolodin — APPROVED** [review]

**Pull request merged**